daemon/wrapnix_test.go

```diff
@@ -0,0 +1,87 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package daemon
+
+import (
+	"gopkg.in/check.v1"
+	"net"
+	"path/filepath"
+	sys "syscall"
+)
+
+type wrapSuite struct {
+	ucred *sys.Ucred
+	err   error
+}
+
+var _ = check.Suite(&wrapSuite{})
+
+func (s *wrapSuite) getUcred(fd, level, opt int) (*sys.Ucred, error) {
+	return s.ucred, s.err
+}
+
+func (s *wrapSuite) SetUpSuite(c *check.C) {
+	getUcred = s.getUcred
+}
+
+func (s *wrapSuite) TearDownTest(c *check.C) {
+	s.ucred = nil
+	s.err = nil
```

Comment:
seems you have s.err available but there's no test setting it and so checking the getUcred returns error path